s3auth-relay/src/test/java/com/s3auth/relay/HttpResponseTest.java

```diff
@@ -170,4 +181,18 @@ public Void call() throws Exception {
         );
     }
 
+    /**
+     * Test for reproducing one of the errors in issue #202.
```

Comment:
please, provide more informative description. What is ti test for? What is expected outcome?